core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/coredns-ocp-dnsnameresolver
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/coredns-ocp-dnsnameresolver
+  review_acts_as_lgtm: true
```

Comment:
That is better to be set to `false` because this allows people not mentioned in the OWNERS file to give approve PRs.